tower-web/src/app/modules/main/component/confirm-delete-dialog/confirm-delete-dialog.component.scss

```diff
@@ -0,0 +1,9 @@
+.actions{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.form-field--full-width{
+  width: 100%;
+}
```

Comment:
Actually these are a bit redundant as we can use bootstrap utility classes `flex-row justify-content-around` and `w-100` for this purpose